.github/workflows/build-and-check-fleetctl-docker-and-deps.yml

```diff
@@ -23,7 +27,7 @@ jobs:
     runs-on: ubuntu-22.04
     environment: Docker Hub
     permissions:
-      contents: write
+      id-token: write # for aws-actions/configure-aws-credentials
```

Comment:
guessing this is ok since the workflow passed, but confirming `contents: write` wasn't needed here?

Reply:
Yeah, probably a copy-paste residue when I created the workflow.